triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -535,7 +535,9 @@ class Runner(
       val f: Future[Empty] = client.commandClient
         .submitSingleCommand(req)
       f.map(_ => None).recover {
-        case s: StatusRuntimeException =>
+        case s: StatusRuntimeException if s.getStatus != io.grpc.Status.UNAUTHENTICATED =>
```

Comment:
Did you manage to hit this? I would expect that our transaction stream gets killed before that but maybe that only happens if a transaction is sent?

Reply:
Yes, without this change the error is caught and injected into the DAML code as a failed completion.